tests/e2e/cucumber/environment/index.ts

```diff
@@ -173,3 +176,10 @@ const cleanUpGroup = async (adminUser: User) => {
   await Promise.all(requests)
   createdGroupStore.clear()
 }
+
+const removeDirectory = () => {
+  const folder_path = path.join(config.assets, '/multipleFiles')
+  if (fs.existsSync(folder_path)) {
+    fs.rmSync(folder_path, { recursive: true })
+  }
+}
```

Comment:
let's have a temp folder where we create files folders during tests in `filesForUpload` and the path in the `config.js`:
```js
// config.js
...
  assets: './tests/e2e/filesForUpload',
  assetsTemp: './tests/e2e/filesForUpload/temp',
...
```
this way, we can easily cleanup the folder `config.assetsTemp`